### PR TITLE
samples: cellular: location: Enable Wi-Fi by default for Thingy91x

### PR DESCRIPTION
**Pull request by trantanen**
Initially location sample had Wi-Fi disabled by default for Thingy91x (PR #14418).
Reverting that decision and aligning with some other apps/samples.

Jira: NCSDK-30108
Jira: NCSDK-30166

### CODE REVIEW

File: samples/cellular/location/README.rst [hunk 1]
```diff
@@ -83,17 +82,8 @@ For example:
...
-To build the Location sample with Thingy:91 X Wi-Fi support, use the ``-DDTC_OVERLAY_FILE=thingy91x_wifi.overlay``, ``-DEXTRA_CONF_FILE=overlay-nrf700x-wifi-scan-only.conf``, ``-DSB_CONFIG_WIFI_NRF70=y``, and ``-DSB_CONFIG_WIFI_NRF70_SCAN_ONLY=y`` options.
-For example:
-
-.. code-block:: console
-
-   west build -p -b thingy91x/nrf9151/ns -- -DDTC_OVERLAY_FILE=thingy91x_wifi.overlay -DEXTRA_CONF_FILE=overlay-nrf700x-wifi-scan-only.conf -DSB_CONFIG_WIFI_NRF70=y -DSB_CONFIG_WIFI_NRF70_SCAN_ONLY=y
-
-See :ref:`cmake_options` for more instructions on how to add these options.
+.. note::
+   Thingy:91 X build has Wi-Fi support enabled by default. You do not need to add any overlays.
```
Comment (peknis) [inline samples/cellular/location/README.rst:86]:
```suggestion
   The Thingy:91 X build supports Wi-Fi by default.
   You do not need to add any overlays.
```

**Review by peknis — CHANGES_REQUESTED** [review]

**Review by simensrostad — APPROVED** [review]
LGTM

**Review by jtguggedal — APPROVED** [review]

**Comment by trantanen** [comment]
Added changelog still.

**Review by tokangas — APPROVED** [review]
Approved with one minor nit, you could use the official device name also in the commit message, i.e. "Thingy:91 X".

**Review by peknis — APPROVED** [review]

**Review by nordicjm — APPROVED** [review]

**Review by jhirsi — APPROVED** [review]

**Pull request merged**